elifetools/xmlio.py

```diff
@@ -7,21 +7,36 @@
 xmlio can do input and output of XML, allowing it to be edited using ElementTree library
 """
 
+class CustomXMLParser(ElementTree.XMLParser):
+    doctype_dict = {}
```

Comment:
the scope of these values is limited to the parser object, which is vastly better than the main Python module